crt_portal/static/js/filters.js

```diff
@@ -272,6 +272,16 @@
     }
   };
 
+  root.CRT.radioButtonView = function(props) {
+    for (var i = 0; i < props.el.length; i++) {
+      props.el[i].addEventListener('change', function(event) {
+        if (event.target.checked) {
+          root.CRT.filterDataModel[props.name] = event.target.value;
+        }
+      });
+    }
```

Comment:
nit: I see this was done for consistency; I'm in favor of following new patterns in these files as we go.

Not fixing the rest of the file, but including new/more readable patterns in new code:

```suggestion
    props.el.forEach(el => 
      el.addEventListener('change', function(event) {
        if (!event.target.checked) continue;
        root.CRT.filterDataModel[props.name] = event.target.value;
      });
    });
```